test.py

```diff
@@ -1,9 +1,13 @@
-import ee;
```

Comment:
AFAICT this whole file is not necessary. can it be removed?

Reply:
I do use it for test. I can add it to git ignore if that suits you better ❤️

Reply:
I'm just trying to understand what it's for. Pytest + tox seem to do the job for me, and less is more, so I'm trying to understand how this is used, and what's the best way to achieve that goal.